src/unix/linux_like/mod.rs

```diff
@@ -1362,6 +1362,16 @@ cfg_if! {
     }
 }
 
+safe_f! {
```

Comment:
We have another `safe_f!` here, the below should be placed there. And we may want to make them `const fn`s.

Reply:
I moved it to the other `safe_f!`, did I get the `const`ness right?